meta/ColorMathInterface.m

```diff
@@ -0,0 +1,422 @@
+(* ::Package:: *)
```

Comment:
Should this file really be part of this pull request?

Reply:
I looked more carefully and it turned out, that this package is used **only** by test of f_to_f_convesion. So I decided to move the code inside it in commit 0b93cdc.